ci/check-documentation.py

```diff
@@ -79,6 +99,17 @@ def complain(message):
                     ' does not exist or is not a file')
                 continue
 
+            if check_md_links and url not in readme_files:
+                readme_files.append(url)
+
+        if check_md_links:
+            for md_file in md_files:
+                # glob uses lowercase filenames on Windows
```

Comment:
Does it? That doesn't seem to be the case:

```
>>> g = Path().glob('**/README.md')
>>> next(g)
WindowsPath('README.md')
```

But, even if it does, `WindowsPath` comparison ignores case on Windows, so as long as you stick with `Path` objects, any comparisons should work anyway.